confgenerator/testdata/invalid/windows/logging-receiver_systemd/golden_error

```diff
@@ -0,0 +1 @@
+logging receiver with type "systemd" is not supported. Supported logging receiver types: [files, nginx_access, nginx_error, syslog, tcp, windows_event_log].
```

Comment:
For naming consistency, could we make the folder name explicit that it's not supported? e.g.

```
confgenerator/testdata/invalid/windows/logging-receiver_unsupported_type_systemd
```

Reply:
Done